test/junk_data.cc

```diff
@@ -28,12 +28,10 @@ int main(int argc, const char *argv[])
     if (argc != 2) {
         return 1;
     }
-    long long count = strtoull(argv[1], NULL, 10);
+    unsigned long long count = strtoull(argv[1], NULL, 10);
```

Comment:
Should the NULL here be changed to nullptr?

Reply:
Done.  I'll fix the clang-tidy target in another PR.